src/test/java/httpServer/ConsoleWriterTest.java

```diff
@@ -0,0 +1,27 @@
+package httpServer;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ConsoleWriterTest {
+
+    @Test
+    void printsHello() {
+        var outputStream = new ByteArrayOutputStream();
+        var printStream = new PrintStream(outputStream);
+        var oldPrintStream = System.out;
+        System.setOut(printStream);
+        var expectedOutput = "Hello\n";
+
+        new ConsoleWriter().println("Hello");
+
+        System.out.flush();
+        System.setOut(oldPrintStream);
+
+        assertEquals(expectedOutput, outputStream.toString());
+    }
```

Comment:
Good to know how to write this test, but in practice I'm not sure how valuable it really is.